gyb_syntax_support/AttributeNodes.py

```diff
@@ -199,6 +201,46 @@
                    '''),
          ]),
 
+    # The argument of '@_package(...)'
+    # package-attr-arguments -> package-location-label ':' package-location
+    #                           ( ',' package-requirement-label? ':'? package-requirement )?
+    #                           ( ',' 'product' ':' package-product-name )?
+    Node('PackageAttributeArguments', name_for_diagnostics='@_package arguemnts',
+         kind='Syntax',
+         description='''
+         The arguments for the `@_package` attribute imitating `PackageDescription`
+         ''',
+         children=[
+             Child('LocationLabel', kind='IdentifierToken',
+                   text_choices=['url', 'path', 'id'],
+                   description='The location label.'),
+             Child('LocationColon', kind='ColonToken'),
+             Child('Location', kind='StringLiteralToken',
+                   description='The location/identifier of package.'),
+             Child('LocReqComma', kind='CommaToken',
```

Comment:
Since it will be the name of the child in the SwiftSyntax API, I would prefer non-abbreviated names like `LocationRequirementComma` or `CommaBetweenLocationAndRequirement` (I think I prefer the latter, don’t have super strong opinions though).

(same below for the next comma)

Reply:
done👌